src/components/ToggleSwitch/index.js

```diff
@@ -17,7 +20,10 @@ ToggleSwitch.propTypes = {
   size: oneOf(['x-small', 'small', 'large']),
 
   /** Color for disabled toggle switch */
-  skin: oneOf(['standard', 'error', 'success'])
+  skin: oneOf(['standard', 'error', 'success']),
+
+  /** Content of the toggle switch */
+  children: any
```

Comment:
Are you getting errors if you don't have this? I think we are getting this by having 
```...CoreToggleSwitch.propTypes,``` above us.

Also, please double check with Ben, maybe in the backoffice we are allowing only string as the label? not sure. If he doesn't answer then we will merge. we will not let it slow us down